apps/i18n/common/en_us.json

```diff
@@ -438,7 +438,7 @@
   "dataTableHeader": "DATA TABLES",
   "dataTableNamePlaceholder": "Table name",
   "dataLibraryHeader": "Data Library",
-  "dataLibraryDescription": "Find and import tables of real data from the Code.org library.",
+  "dataLibraryDescription": "Tables of real data for your app. Help us [add more data](https://support.code.org/hc/en-us/articles/360044953271-Help-us-add-new-data-sets-into-App-Lab-).",
```

Comment:
The startWebRequest doc says "Want to use a URL that's not currently allowed? Let us know at support@code.org.", which I feel like reads really nicely as an invitation to contribute. What about something like "Want to use a dataset not listed here? [Help us add more datasets](link)"

Reply:
agreed - done!